subsys/bluetooth/controller/Kconfig

```diff
@@ -749,7 +749,7 @@ config BT_CTLR_ADV_ISO_STREAM_COUNT
 config BT_CTLR_ADV_ISO_PDU_LEN_MAX
 	int "Maximum Broadcast Isochronous Channel PDU Length"
 	depends on BT_CTLR_ADV_ISO
-	range 27 251
+	range 1 251
```

Comment:
Do you think it is a bug in the HCI spec that it allows PDU size of 0 with the ISO test commands?

Reply:
No, it is fine in the Spec as it is applicable for Connected ISO where one direction can have Empty ISO PDUs. 

Here this is Broadcast ISO where (ex, LE Create BIG Test Command) Max_PDU is 0x0001 to 0x00FB.

Reply:
Thanks :) I hadn't caught that minimum PDU between connected and broadcast was different (fixed that in my PR that uses the test parameters)

Reply:
That said controller defines `BT_CTLR_CONN_ISO_PDU_LEN_MAX`  to `range 1 251`  because this Kconfig is applicable for both directions and one direction atleast has to have 1 byte PDU else makes no sense for an application.

Reply:
Agreed